File: backend/src/main/kotlin/com/example/backend/dtos/SortieDto.kt
package com.example.backend.dtos

import java.math.BigDecimal
import java.time.LocalDateTime


data class SortieDetailDto(
  val enrayon: List<Enrayon>?,
  val produitDetailId: Int?,
  val typeSortieId: Int?,
)

data class Enrayon(
  val contenuDetail: String?,
  val id: Int?,
  val quantite: Int?,
  val rayonId: String?,
  val stockTotal: Int?
)

File: backend_py/pharmaPython/app/schemas/sortie_dto.py
from pydantic import BaseModel
from typing import Optional, List

class Enrayon(BaseModel):
  contenuDetail: Optional[str] = None
  id: Optional[int] = None
  quantite: Optional[int] = None
  rayonId: Optional[str] = None
  stockTotal: Optional[int] = None

class SortieDetailDto(BaseModel):
  enrayon: Optional[List[Enrayon]] = None
  produitDetailId: Optional[int] = None
  typeSortieId: Optional[int] = None
